README.md

```diff
@@ -33,9 +33,11 @@ supported versions of Kubernetes clusters.
 
 #### Compatibility matrix
 
-|                  | Kubernetes 1.10 |
-|------------------|-----------------|
+|                  | Kubernetes 1.10 | Kubernetes 1.13 |
+|------------------|-----------------|-----------------|
 | client 0.1.0a1   | ✓               |
+|------------------|-----------------|-----------------|
```

Comment:
when click on 'View file', i see  -------------------- row in the matrix, is it intentional?

Reply:
fixed.